waku/v2/node/keepalive.go

```diff
@@ -19,6 +22,8 @@ const maxAllowedPingFailures = 2
 // the peers if they don't reply back
 const sleepDetectionIntervalFactor = 3
 
+const maxPeersToPing = 10
+
 // startKeepAlive creates a go routine that periodically pings connected peers.
 // This is necessary because TCP connections are automatically closed due to inactivity,
```

Comment:
The keepalive code was added to match the behavior seen here:
https://github.com/waku-org/nwaku/blob/master/waku/node/waku_node.nim#L1155-L1172

Which as far as i could see was introduced due to this issue: https://github.com/waku-org/nwaku/issues/504
and fixed in https://github.com/waku-org/nwaku/pull/523 with the usage of GRAFT messages, but eventually replaced by Ping protocol in https://github.com/waku-org/nwaku/pull/621/f

Reply:
I think so yes!

With this PR, when using go-waku as service node we do ping all mesh peers every 5m,while in status-go, we do so every 10s.

Reply:
> I think so yes!
> 
> With this PR, when using go-waku as service node we do ping all mesh peers every 5m,while in status-go, we do so every 10s.

But with this PR in case of status-go as well we don't need to ping all peers every 10 secs is what i am thinking. Ping all peers once in 5min...and only ping 5 peers or so every 10 secs just to know that we are not disconnected from the network(This should be the logic in combined go-waku+status-go when acting as relay+serviceNode).

Reply:
:thinking: 
Let's try that out!

Reply:
Check latest commit and lmk what you think. Do note that I hardcoded 10s for both using go-waku as a service node as well as the c-bindings instead of creating flags, because those are going to be replaced by nwaku anyway.